scripts/benchmark.js

```diff
@@ -128,6 +130,9 @@ class Benchmark {
     const out = compile(flags);
     return out.compiledCode;
   }
+  butternut(code) {
+    return butternut.squash(code, { check: true }).code;
```

Comment:
Using the `check: true` option is a good idea to check that Butternut is producing valid code, but it *shouldn't* be used for benchmarking as it causes the output to be parsed again.

Also, Butternut generates a sourcemap by default. Since Babili and Uglify don't, we should probably disable sourcemaps for Butternut too to remove that from the equation:

```diff
-return butternut.squash(code, { check: true }).code;
+return butternut.squash(code, { sourceMap: false }).code;
```